docs/client_reference.rst

```diff
@@ -30,8 +30,8 @@ Usage example::
 
 
 .. class:: ClientSession(*, connector=None, loop=None, request_class=None,\
-                          response_class=None, cookies=None, headers=None,\
-                          auth=None)
+                          response_class=None, ws_response_class=None,\
+                          cookies=None, headers=None, auth=None)
```

Comment:
Documentation is wrong.
I've changed defaults but forgot to update doc.
Please fix to correct values.